R-package/tests/testthat/test_basic.R

```diff
@@ -380,6 +380,30 @@ test_that("cv works", {
   expect_false(is.null(bst$record_evals))
 })
 
+test_that("CVBooster$reset_parameter() works as expected", {
+  dtrain <- lgb.Dataset(train$data, label = train$label)
+  cv_bst <- lgb.cv(
+    params = list(
+      objective = "regression"
+      , min_data = 1L
+      , num_leaves = 7L
+      , verbose = VERBOSITY
+    )
+    , data = dtrain
+    , nrounds = 2L
+    , nfold = 2L
+  )
+  expect_is(cv_bst, "lgb.CVBooster")
+  expect_length(cv_bst$boosters, 2L)
```

Comment:
Please use different values for `nrounds` and `nfold` arguments to make it clearer what value is asserted in `expect_length(cv_bst$booster, ...`.

Reply:
ah yeah, good idea!

updated in https://github.com/microsoft/LightGBM/pull/4901/commits/52bce82c4a7ad8b13e531956b7f23f3cadb9b02c